[front] - chore: extend SearchResultOutput

JulesBelveze opened the pull request:
## Description

This PR:
- Adds optional `data_source_id` and `data_source_view_id` to the search result resource schema
- Populates new schema fields with the respective IDs in the search function output

## Tests

Locally

## Risk

Low, field is optional

## Deploy Plan

Deploy front

JulesBelveze committed:
[front/lib/actions/mcp_internal_actions] - feature: extend SearchResultResourceSchema with data source identifiers

 - Add optional `data_source_id` and `data_source_view_id` to the search result resource schema
 - Populate new schema fields with the respective IDs in the search function output

aubin-tchoi reviewed (approved):
LGTM, I think you also need to add it in the data source filesystem (also has a search)

JulesBelveze committed:
[front/lib/actions/mcp_internal_actions/tools/data_sources_file_system] - feature: enhance search result metadata

 - Include data source id and data source view id in search result source information
